.github/workflows/scripts/linux/flatpak/net.pcsx2.PCSX2.json

```diff
@@ -19,7 +19,6 @@
     "--device=all",
     "--share=network",
     "--share=ipc",
-    "--socket=x11",
```

Comment:
I don't _think_ this is correct
Should we just be reverting the in this commit? https://github.com/PCSX2/pcsx2/pull/10179/commits/033f9c19739c3903b23d121f57581fa4edd58411

Reply:
![image](https://github.com/user-attachments/assets/4a9c00f7-4c07-4715-9d55-34ef6cdbba71)
Yup, you're right, adding socket fallback